docs/sphinx/formats/micro-manager.txt

```diff
@@ -49,6 +49,7 @@ Utility: |Good|
 
 **Additional Information**
```

Comment:
I think this file who was hand-edited. This extra line was generated by the autogen components from the metadata in `format-pages.txt`. It should be fine as it is.